crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -461,39 +461,18 @@ impl BuildSpec {
 	fn build(self, cli: &mut impl cli::traits::Cli) -> anyhow::Result<&'static str> {
 		cli.intro("Building your chain spec")?;
 		let mut generated_files = vec![];
-		let BuildSpec {
-			ref output_file,
-			ref profile,
-			id,
-			default_bootnode,
-			ref chain,
-			genesis_state,
-			genesis_code,
-			..
-		} = self;
+		let BuildSpec { ref output_file, ref profile, id, genesis_code, genesis_state, .. } = self;
 		// Ensure binary is built.
 		let binary_path = ensure_binary_exists(cli, profile)?;
 		let spinner = spinner();
-		spinner.start("Generating chain specification...");
 
-		// Generate chain spec.
-		generate_plain_chain_spec(&binary_path, output_file, default_bootnode, chain)?;
```

Comment:
Moved to generate chain spec and raw chain spec into the `generate_chain_spec` function